README.md

```diff
@@ -36,18 +36,19 @@ In your `eslint.config.js` file, import `@eslint/css` and include the recommende
 
 ```js
 // eslint.config.js
+import { defineConfig } from "eslint/config";
 import css from "@eslint/css";
 
-export default [
+export default defineConfig([
 	// lint CSS files
 	{
 		files: ["**/*.css"],
 		language: "css/css",
-		...css.configs.recommended,
+		extends: ["css/recommended"],
 	},
 
 	// your other configs here
-];
+]);
```

Comment:
This results in `TypeError: Plugin "css" not found.`.

For `extends` from a plugin config to work, we'd need to specify the plugin in the same config object where `extends` is.

Reply:
Doh, right.